Update install_formulas.rb

alex-slynko opened the pull request:
Allow to pass custom options to brew packages. I.e. `brew install vim --with-lua`

chef-supermarket commented:
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @alex-slynko

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

tas50 commented:
There's a rubocop failure showing up in travis for this.  Can you fix the trailing whitespace.

chef-supermarket commented:
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

alex-slynko commented:
@tas50 I fixed the rubocop issue. Added documentation.


Additionally I changed options to be a string, like it is now in [Chef resource](https://docs.chef.io/resource_homebrew_package.html)

alex-slynko commented:
@tas50 Could you please check this again?

tas50 commented:
Thanks @alex-slynko
